core/pom.xml

```diff
@@ -531,6 +531,35 @@ limitations under the License.
         <targetPath>codegen</targetPath>
       </resource>
     </resources>
+
+  <pluginManagement>
+    <plugins>
+    <plugin>
+      <groupId>org.apache.maven.plugins</groupId>
+      <artifactId>maven-javadoc-plugin</artifactId>
+      <!-- This is the configuration used by "mvn javadoc:javadoc". It is
+           configured strict, so that it shows errors such as broken links in
+           javadoc on private methods. The configuration for "mvn site" is
+           under "reporting", and is more lenient. -->
+      <configuration>
+        <sourceFileIncludes>${project.basedir}/target/generated-sources/resgen/org/apache/calcite/runtime/Resources.java</sourceFileIncludes>
```

Comment:
Could you please more context on including this file `Resources.java`?

Reply:
When doing "mvn javadoc:jar" in calcite/core, I got many error like:
"Error while generating javadoc: 
  error: cannot find symbol
  import org.apache.calcite.runtime.Resources;"
Because this "Resources.java" file is not found in the sourcepath or classpath.  Adding it to <sourceFileIncludes> resolved these errors.

Reply:
@wmoustafa You are right, I found the javadoc generated by this solution is not complete. I changed it in #23, please review.